core/iwasm/libraries/wasi-nn/src/wasi_nn_tensorflowlite.cpp

```diff
@@ -206,6 +210,17 @@ tensorflowlite_init_execution_context(void *tflite_ctx, graph g,
                 NN_ERR_PRINTF("Error when enabling GPU delegate.");
                 use_default = true;
             }
+#elif defined(WASI_NN_ENABLE_EXTERNAL_DELEGATE)
+            NN_WARN_PRINTF("external delegation enabled.");
+            auto options = TfLiteExternalDelegateOptionsDefault(
+                "/usr/lib/libvx_delegate.so");
```

Comment:
Can we make this parameterizable? Other devices will also use the external delegate functionality.

Reply:
Add compile option as below